src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -703,7 +703,7 @@ export class CellModel extends Disposable implements ICellModel {
 				if (this._outputs.length > 0) {
 					for (let i = 0; i < this._outputs.length; i++) {
 						if (this._outputs[i].output_type === 'execute_result') {
-							this._outputs.splice(i, 0, this.rewriteOutputUrls(output));
+							this._outputs.push(this.rewriteOutputUrls(output));
```

Comment:
@lucyzhang929 looks like you touched this part of the code last. Will changing to `.push()` break streaming?

Reply:
Shouldn't break streaming - I only used splice to show the messages before the table. In fact, if we want the table to show first, lines 702-712 can be removed. The messages will just be added here: https://github.com/microsoft/azuredatastudio/blob/f96fd911c1d0d883587748de8560ffc36d17a17d/src%2Fsql%2Fworkbench%2Fservices%2Fnotebook%2Fbrowser%2Fmodels%2Fcell.ts#L747

Reply:
I have removes the line from 702 -712

Reply:
reverted them back